### DISCUSSION TITLE
boss boundary setter not working with Mania's screen resolution

### DISCUSSION THREAD

**Discussion by UndertaleFan6** [Techokami/SonicWorldsNext#200] [Q&A] [answered]
I'm trying to modify the engine to be a bit more like Mania including the screen size being the same as Mania's. For some reason the screen bounds stop working when you go too much to the left and then you can go anywhere.

**Comment by Sunclass** [comment]
That's actually because of the boundary setter. When you touch the boss's boundary setter, the default screensize is too small to let the player (sonic) touch the boundary setter before the boss. When you increase the screensize to widescreen, the player is able to touch the boundary setter to the left. To fix that, you just need to push the left boundary setter to the left (i'd say about 48-52 px will do it)

**Reply by UndertaleFan6** [reply]
what left boundary setter? there's just one, just like in the "base zone" example level

**Reply by Sunclass** [reply]
![Boundry](https://github.com/user-attachments/assets/cf141cbe-3a75-47e0-86ab-80b728094db5)
these two are just an example. if you touch boundrysetter2, you can go anywhere in the floor, you can even ignore the boss. Unless i'm mistaken and didn't understand your question correctly

**Reply by UndertaleFan6** [reply]
oh ok now I get it. so what am I supposed to do?

**Reply by Sunclass** [reply, marked as answer]
you can either change the boundrysetter 2's boundary values or just push the boundrysetter 2's position more to the left so that when you activate the boss's boundrysetter, the player can't activate boundrysetter2
(You click boundrysetter 2, go to position and decrement it with 52 px, should do the trick)

**Reply by UndertaleFan6** [reply]
thanks!
